tools/merge_strings_xml.py

```diff
@@ -98,16 +98,23 @@ def merge_strings(main_xml, extra_sections):
         add_section(main_root, insertion_point_index, section_name, new_elements)
     # make sure xml file ends with a newline
     main_root.tail = '\n'
-    xml_string = ('<?xml version="1.0" encoding="UTF-8"?>' + '\n' + 
+    xml_string = ('<?xml version="1.0" encoding="UTF-8"?>' + '\n' +
         ET.tostring(main_root, encoding='utf8').split('\n',1)[1])
     with open(main_xml, 'wb') as xml_file:
         xml_file.write(xml_string)
 
 def main():
+    if len(sys.argv) != 3:
+        print 'Usage: merge_strings_xml.py <section name> <strings.xml file path>'
+        sys.exit(1)
+
```

Comment:
Nitpick. What do you think of assigning the argv elements to variables after this?

```python
section_name = sys.argv[1]
input_file = sys.argv[2]
```

This way we don't need to repeat argv accesses and readers won't need to scroll back to the usage text to know what `argv[x]` is.